airbyte-integrations/connectors/destination-deepset/pyproject.toml

```
@@ -0,0 +1,28 @@
+[build-system]
+requires = ["poetry-core>=1.0.0"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+version = "0.1.0"
+name = "destination-deepset"
+description = "Destination implementation for deepset."
+authors = ["Airbyte <contact@airbyte.io>"]
+license = "MIT"
+readme = "README.md"
+documentation = "https://docs.airbyte.com/integrations/destinations/deepset"
+homepage = "https://airbyte.com"
+repository = "https://github.com/airbytehq/airbyte"
+packages = [{ include = "destination_deepset" }, { include = "main.py" }]
+
+[tool.poetry.dependencies]
+python = "^3.9,<3.12"
+airbyte-cdk = "^0"
```

```suggestion
airbyte-cdk = "^6"
```

Dammit I should really update the template project.

In theory, you will get three compatibility as most things have not changed for destinations, BUT Airbyte protocol models did change significantly and switched from Pydantic to Dataclasses.

It sucks if this made you spend time on something that will need to be updated, but I'd really like this to start form a new CDK on day one.